addons/modules/functions/fnc_moduleHideTerrainObjects.sqf

```diff
@@ -0,0 +1,55 @@
+#include "script_component.hpp"
+/*
+ * Author: Ampersand
+ * Zeus module function to hide terrain objects.
+ *
+ * Arguments:
+ * 0: Logic <OBJECT>
+ *
+ * Return Value:
+ * None
+ *
+ * Example:
+ * [LOGIC] call zen_modules_fnc_moduleHideTerrainObjects
+ *
+ * Public: No
+ */
+
+params ["_logic"];
+
+private _position = ASLToAGL getPosASL _logic;
+deleteVehicle _logic;
+
+["STR_a3_to_hideTerrainObjects1", [
+    ["TOOLBOX:YESNO", "STR_a3_to_hideTerrainObjects1", true],
+    ["EDIT", ELSTRING(common,Range), 10],
+    ["CHECKBOX", "STR_a3_to_hideTerrainObjects6", true], // Buildings
+    ["CHECKBOX", "STR_a3_to_hideTerrainObjects7", true], // Walls & Fences
+    ["CHECKBOX", "STR_a3_to_hideTerrainObjects8", true], // Vegetation
+    ["CHECKBOX", "STR_a3_to_hideTerrainObjects9", true] // Other
+], {
+    params ["_values", "_position"];
+    _values params ["_hide", "_range", "_hideBuildings", "_hideWalls", "_hidePlants", "_hideOthers"];
```

Comment:
Use vegetation in the variable name instead of plants to match display name. 

Also, I think `_includeBuildings`, `_includeWalls, ... would be better variable names since these don't specify whether they are hidden but rather if they are included in the type filter.

Reply:
6dd147a
a296bce